app/src/common/shared/com/igalia/wolvic/ui/views/library/BookmarksView.java

```diff
@@ -429,7 +429,7 @@ protected void onLayout(boolean changed, int left, int top, int right, int botto
     protected void updateLayout() {
         post(() -> {
             double width = Math.ceil(getWidth()/getContext().getResources().getDisplayMetrics().density);
-            boolean isNarrow = width < SettingsStore.WINDOW_WIDTH_DEFAULT;
+            boolean isNarrow = width < SettingsStore.getInstance(getContext()).getWindowWidth();
```

Comment:
This change is not needed. The size that triggers the narrow layout needs to be predictable.